From e1f47d3065d08c4c02ce717745c6416fb402869b Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Tue, 27 Sep 2022 16:13:19 +0800
Subject: [PATCH] s32g: Aptiv: set correct path for DDR files

Since DDR files are all in drivers/nxp/s32/ddr/aptiv_cvc_sousa/ but not
drivers/nxp/s32/ddr/s32g2, we need set correct path for them.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 plat/nxp/s32/s32_ddr.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/nxp/s32/s32_ddr.mk b/plat/nxp/s32/s32_ddr.mk
index 1f7f97dc9..14ef059ff 100644
--- a/plat/nxp/s32/s32_ddr.mk
+++ b/plat/nxp/s32/s32_ddr.mk
@@ -12,7 +12,7 @@ DDR_UTILS_FILE	:= ${CUSTOM_DDR_DRV}/ddr_utils.h
 PLAT_DDR_DRV	:= ${CUSTOM_DDR_DRV}
 else
 COMMON_DDR_DRV	= ${DDR_DRV}
-PLAT_DDR_DRV	= ${DDR_DRV}/${S32_PLAT_SOC}
+PLAT_DDR_DRV	= ${DDR_DRV}/aptiv_cvc_sousa
 endif
 
 DDR_DRV_SRCS += \
@@ -26,7 +26,7 @@ DDR_DRV_SRCS += \
 	${COMMON_DDR_DRV}/ddr_lp.c \
 	${COMMON_DDR_DRV}/ddr_lp_csr.c \
 	${COMMON_DDR_DRV}/ddrss_cfg.c \
-	${COMMON_DDR_DRV}/imem_cfg.c \
+	${PLAT_DDR_DRV}/imem_cfg.c \
 	${DDR_DRV}/ddr_density.c \
 
 # If CUSTOM_DDR_DRV is set, this target modifies the ddr_utils.h file
-- 
2.36.1

